runtime/android/core_shell/src/org/xwalk/core/xwview/shell/XWalkViewShellActivity.java

```diff
@@ -109,6 +110,7 @@ private void unregisterTracingReceiver() {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        XWalkCoreWrapper.initEmbeddedMode();
```

Comment:
I suggest to add a FIXME or TODO here to remove this unexpected call of internal methods.